packages/grafana-ui/src/slate-plugins/preventNewLine.ts

```diff
@@ -0,0 +1,20 @@
+import { Plugin } from '@grafana/slate-react';
+import { Editor as CoreEditor } from 'slate';
+
+// This plugin should be used to prevent inserting new lines when pressing shift+enter inside a query editor
+export function PreventNewLineOnShiftEnterPlugin({ handler }: any): Plugin {
```

Comment:
This needs a release tag to pass the fe docs test
```
Warning: dist/slate-plugins/preventNewLine.d.ts:2:1 - (ae-missing-release-tag) "PreventNewLineOnShiftEnterPlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Reply:
Oh okay, which release tag do you think it needs? I'm guessing `@alpha` but i've never used a release tag before.

Reply:
I think @alpha or @internal would be good